### DISCUSSION TITLE
Calendar as Card

### DISCUSSION THREAD

**Discussion by Nikolai-Kaskatiiski** [wdelfuego/nova-calendar#65] [Ideas & Feature Requests]
I have just started to use this tool. Looks amazing to  me. I would like to have  it on my dashboard, is it possible to exist as a card?

**Comment by wdelfuego** [comment]
This is not currently possible, but it'd be a nice feature to have!

Issue is related to #64, showing the Calendar on resource Detail pages instead of only as an independent Tool.

This is not something I can work on in the coming weeks or months but I might pick it up at some point in the future. 
There's always the possibility to [sponsor](https://github.com/sponsors/wdelfuego?frequency=one-time&sponsor=wdelfuego) the development of this feature too, if you'd like me to work on it sooner.

If anyone is interested in working on this, get in touch here :)!
